pkg/matcher/matcher.go

```diff
@@ -39,6 +66,33 @@ func (w *allowlistMatcher) Match(fullPath string, _ os.FileInfo, _ io.ReaderAt)
 	return false, false
 }
 
+func (w *allowlistMatcher) GetCommonPrefixDirs() []string {
```

Comment:
nit: w *allowlistMatcher, w should be a or m?

Reply:
:+1: